### PR TITLE
Update path-too-long-exception.md

### PR DESCRIPTION
**Pull request by Abdjulaziz**
## Description
Changed Database query

_What did you add/update/change?_
Changed database query to show media items longer than 80 characters. Previous query showed all media now you only get the media you need to edit.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_
As soon as possible

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @Abdjulaziz ! Makes sense to change this query 💪 

Reading the steps that comes after running the query.. would it make sense to update those as well?
They talk about identifying the files with the long name 🤔 I guess that won't be necessary with this new script?

What do you think?

**Comment by Abdjulaziz** [comment]
I'll look into that and fix it 👍🏻

Just noticed there's a little bit more that can be improved

**Comment by Abdjulaziz** [comment]
@sofietoft had to test something and ask internally to see if the steps were correct and they were so I did a small change to the text and removed some things

File: umbraco-cloud/troubleshooting/deployments/path-too-long-exception.md [hunk 1]
```diff
@@ -55,18 +55,18 @@ In order to fix this error, you will need to connect to the Live Environments pr
...
+    ORDER BY LEN(path) DESC;
     ```
-3. Identify the files with a path longer than 80 characters.
-4. Find the files in the Media section in the Umbraco backoffice.
-5. Remove the media files from the backoffice.
+3. Find the files in the Media section in the Umbraco backoffice.
+4. Remove the media files from the backoffice.
    * Make sure to note down where the media item is being used in the content
-6. Change the names of the media files giving them shorter names.
+6. Change the names of the media files giving them a name less than 80 characters.
```
Comment (sofietoft) [inline umbraco-cloud/troubleshooting/deployments/path-too-long-exception.md:69]:
```suggestion
6. Change the names of the media files giving them a name shorter than 80 characters.
```

**Review by sofietoft — COMMENTED** [review]
Looks great @Abdjulaziz ! 💪 

I just have a single tiny suggestion.

**Comment by Abdjulaziz** [comment]
Sound good

**Comment by sofietoft** [comment]
Thanks again for the PR @Abdjulaziz ! 💪 

I'll get it merged.

**Pull request merged**